controllers/rhmi/rhmi_controller.go

```diff
@@ -231,6 +232,49 @@ func (r *RHMIReconciler) Reconcile(request ctrl.Request) (ctrl.Result, error) {
 		installationCfgMap = installation.Spec.NamespacePrefix + DefaultInstallationConfigMapName
 	}
 
+	skuSecretName := ""
+	installationSKU := &sku.SKU{}
+	if installation.Spec.Type == string(rhmiv1alpha1.InstallationTypeManagedApi) {
+		skuSecretName, found, err := addon.GetStringParameterByInstallType(context.TODO(), r.Client, rhmiv1alpha1.InstallationTypeManagedApi, request.NamespacedName.Namespace, "sku")
```

Comment:
Hi @sergioifg94 Thanks for the review. in the meantime I've moved the skuSecretName inside the if. It's only needed in there because the name gets assigned in the installationSKU object so should be referred to from there. But it's a good point which I didn't spot. 👍